pkgs/applications/networking/instant-messengers/electronim/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, electron_25
+, python3
+}:
+let
+  electron = electron_25;
+in
+buildNpmPackage rec {
+  pname = "electronim";
+  version = "0.0.95";
+
+  src = fetchFromGitHub {
+    owner = "manusa";
+    repo = "electronim";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DNEhptYu9JV0jQw3q4BIEdPiIHejHei1r7gznvKL37I=";
+  };
+
+  patches = [ ./remove-electron-package-json.patch ];
+
+  postPatch = ''
+    cp ${./package-lock.json} package-lock.json
```

Comment:
Including a huge `package-lock.json` in nixpkgs is considered unacceptable by some committers. Could you please try upstreaming it?

Reply:
I'll try

Reply:
I came up with a better solution, not requiring a changed package-lock.json
I might also upstream this patch later